src/reducers/books.js

```diff
@@ -0,0 +1,45 @@
+const initialBookState = {
+  books: [
```

Comment:
### Optional 
These books should be actually be created in `src/index.js`
![image](https://user-images.githubusercontent.com/31897434/109641061-8f333f80-7b51-11eb-908b-c8aec6cdc195.png)